python-main.js

```diff
@@ -1508,11 +1520,11 @@ function web_editor(config) {
 
     function modalMsg(title, content, links){
         var overlayContainer = "#modal-msg-overlay-container";
-        $(overlayContainer).css("display","block");
+        $(overlayContainer).css("display", "block");
         $("#modal-msg-title").text(title);
         $("#modal-msg-content").html(content); 
+        var modalLinks = [];
```

Comment:
If `$("#modal-msg-links").html((modalLinks).join(' | '));` went inside the if statement again, this can go back as well.
And if we remote the extra white line at 1537 and the extra space from line 1523 (although I agree that there should be spaces after comas), then the diff is smaller and this functions isn't affected.

Reply:
Right, yes, if the `$("#modal-msg-links").html((modalLinks).join(' | '));` goes outside, this variable declaration has to as well. Cool, thanks for the clarification.